### ISSUE TITLE
screen drawing in the tax office

### ISSUE THREAD

**Issue by hymie0** [atsb/RL_M#23]
I went to pay off some of my taxes...

After I paid as much as I had, the top of the screen did not update

```
Welcome to the Larn Revenue Service district office.  How can we help you?

You presently owe 3429 gp in taxes.  

You have   2057 gp.    
```

Even though the bottom of the screen said

```
How much? *You presently owe 1372 gp in taxes. You have no gold pieces.  
```

**Comment by atsb** [comment]
I'll work on this tonight, these last few bugs will be fixed before 13.4

**Comment by atsb** [comment]
Fixed in latest commit.
